.storybook/webpack.config.js

```diff
@@ -15,18 +13,15 @@ module.exports = ({ config }) => {
                         prettierConfig: {
                             printWidth: 150,
                             tabWidth: 5,
```

Comment:
Tabwidth 5 seems rather unusual to me. Is that correct?

Reply:
Good eye. Fixed.